bgpd/bgp_attr.c

```diff
@@ -2234,6 +2234,20 @@ static bgp_attr_parse_ret_t bgp_attr_psid_sub(int32_t type,
 		}
 	}
 
+	/*
+	 * Placeholder code for Unsupported TLV
+	 *  - SRv6 L3 Service TLV (type5)
+	 *  - SRv6 L2 Service TLV (type6)
+	 */
+	else if (type == BGP_PREFIX_SID_SRV6_L3_SERVICE
+	    || type == BGP_PREFIX_SID_SRV6_L2_SERVICE) {
+		flog_warn(
```

Comment:
Do we really want to fill up the log file with possibly 10's of thousands of lines about unsuportted prefix SID's?  I would prefer changing this to a debug and guarding it to only see when debugs are turned on.

Reply:
Thank you reviewing so quickly. (I'm very happy :) )

Currently, there are some Prefix-SID's sub-type (BGP_PREFIX_SID_XXX) by newer I-D.
So, If bgpd connects to newer Cisco's SRv6 aware Router, bgpd fill up the log file with 10's of thousands of line as you say

So, I fix the commit soon.